cider/exec/nextgen/jitlib/llvmjit/LLVMJITValue.cpp

```diff
@@ -317,6 +317,45 @@ JITValuePointer LLVMJITValue::castPointerSubType(JITTypeTag type_tag) {
       JITTypeTag::POINTER, parent_function_, new_llvm_value, "cast_ptr", false, type_tag);
 }
 
+JITValuePointer LLVMJITValue::castJITValuePrimitiveType(JITTypeTag target_jit_tag,
+                                                        bool is_signed) {
```

Comment:
Seems ```is_signed``` could be get from ```source_type```. It's unnecessary to pass it as a parameter.

Reply:
1. so need to keep this parameter in the `JITValue` method for other compiler impl?  now already set it as a default parameter in `LLVMJITValue.h` file.
2. ` getLLVMType` method will get `llvm::Type` from `JITTypeTag`, seems the specified default type is signed, so can just remove unsigned-cast part?

Reply:
- I think needn't. Because we can get all type attributes (signed, float, width...) from ```JITTypeTag``` itself, by some helper functions.
 - Currently unsigned integer aren't considered as primitive type in jitlib to keep alignment with original omnisci's type system. We could support unsigned integer easily if needed.